assets/js/modules/pagespeed-insights/dashboard/dashboard-widget-homepage-speed-column.js

```diff
@@ -43,8 +44,8 @@ import {
 	PageSpeedReportScoreGauge,
 } from './util';
 
-const isZeroData = ( data ) => {
-	return 0 === data.categories.performance.score;
+const hasData = ( data ) => {
```

Comment:
Not sure about this renaming in regards to consistency (the `isZeroData` naming is used throughout many places). But this part of the codebase is not really consistent anyway, and I like your new name better, so let's go with it.